src/libsync/propagateremotemkdir.cpp

```diff
@@ -162,10 +162,14 @@ void PropagateRemoteMkdir::finalizeMkColJob(QNetworkReply::NetworkError err, con
             job->start();
         }
     });
-    connect(propfindJob, &PropfindJob::finishedWithError, this, [this]{
+    connect(propfindJob, &PropfindJob::finishedWithError, this, [this] (QNetworkReply *reply) {
+        auto err = QNetworkReply::NetworkError::UnknownNetworkError;
+        if (reply) {
```

Comment:
@mgallien Can't you just go with `const auto err = reply ? reply->error() : QNetworkReply::NetworkError::UnknownNetworkError;`